### DISCUSSION TITLE
Where are repositories stored?

### DISCUSSION THREAD

**Discussion by henriiquecampos** [GetPublii/Publii#1903] [Help]
Hey there, first of all, love Publii.

Secondly, where I can find the Git repositories used to sync the site? I'm getting this error, and I think I need to just pull the changes I did recently in the repo.

![image](https://github.com/user-attachments/assets/2cdce36f-84b6-4946-9a74-45845c79d298)

**Comment by dziudek** [comment]
@henriiquecampos - if you have made changes on the repository then definitely it will break sync. Because the git integration in Publii is one-way - we have added it just for two reasons - to increase supported deployment platforms and to speed up sync process (git is way faster than Github/Gitlab API).

You can try to sync te repository generated by Publii, but to be honest - I do not know if it will solve your issues and also if it won't create any side effects.

The repository should be created inside the `output` directory
